staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -42,10 +42,15 @@ import (
 type AdmissionOptions struct {
 	// RecommendedPluginOrder holds an ordered list of plugin names we recommend to use by default
 	RecommendedPluginOrder []string
-	// DefaultOffPlugins a list of plugin names that should be disabled by default
-	DefaultOffPlugins []string
```

Comment:
how can one disable a plugin ?
by removing the plugin from DefaultEnabledPluginSet list ?

Reply:
this flag `--disable-admission-plugin`

Reply:
but as a developer, let's say that I have a plugin that I know is in alpha. I want to have it disabled by default. What do I do ? I have to register the plugin and put it into the `RecommendedPluginOrder` list, that makes the plugin to be `off`

Later on when a user comes in and wants to experiment with the plugin, he has to use `--enable-admission-plugin` to make the plugin to be `on`, is that correct ?

Reply:
Just don't enable it by default, and put it into RecommendedOrder.

Reply:
I prefer `DefaultOff` as well since this list will be smaller and answers the question for cluster-admins about, "what else can turn on" which is more common than "what else can I turn off".

Reply:
Makes sense.